configs/frenet_optimal_trajectory_planner.conf

```diff
@@ -1,6 +1,3 @@
-######### Carla config #########
---carla_num_people=0
---carla_num_vehicles=100
```

Comment:
Why are you removing the vehicle & people setup?

Reply:
I added this for an experiment and meant to leave it out of the previous PR.